src/osp/tasks/builder.h

```diff
@@ -86,9 +93,10 @@ class TaskBuilder
             return *this;
         }
 
-        TaskTags::Task const m_taskId;
-        TaskTags &m_rTags;
-        DATA_T &m_rData;
+        TaskId const    m_taskId;
+        Tags            &m_rTags;
```

Comment:
please put a space between the & and the variable.
I almost missed that these were references.